lib/absinthe/phoenix/channel.ex

```diff
@@ -87,22 +90,67 @@ defmodule Absinthe.Phoenix.Channel do
     {:reply, {:ok, %{subscriptionId: doc_id}}, socket}
   end
 
+  def handle_info(
+        %Phoenix.Socket.Broadcast{payload: %{result: %{ordinal: ordinal}}} = msg,
+        socket
+      )
+      when not is_nil(ordinal) do
```

Comment:
Do we want to guard `socket` too to ensure not `nil` or does that get checked upstream?

Reply:
Yeah, I think thats right and that it is checked upstream. Not 100% tho